ftw/news/tests/utils.py

```diff
@@ -0,0 +1,24 @@
+from ftw.simplelayout.interfaces import IPageConfiguration
+from plone.uuid.interfaces import IUUID
+import transaction
+
+
+def create_page_state(obj, block):
+    page_state = {
+        "default": [
+            {
+                "cols": [
+                    {
+                        "blocks": [
+                            {
+                                "uid": IUUID(block)
+                            }
```

Comment:
It would be nice to have this in the ftw.simplelayout builders sometime..
e.g.:
```python
create(Builder('sl content page')
       .with_layout(Builder('sl one column layout')
                    .with_blocks(Builder('sl textblock')))
       .with_layout(Builder('sl two column layout')
                    .with_left_column_blocks(Builder('sl textblock'),
                                             Builder('sl textblock'))))
```
/cc @maethu